lib/mengpaneel/tracker.rb

```diff
@@ -90,6 +90,15 @@ def update(message)
 
         super(message)
       end
+
+      # mixpanel-ruby only handles hash, whereas Javascript handles string and hash.
+      def increment(*args)
+        if args.first.is_a?(Hash)
+          super(tracker.distinct_id, *args)
+        elsif args.first.is_a?(String)
+          super(tracker.distinct_id, args[0] => args[1] || 1)
+        end
```

Comment:
Can this be turned into a `case args.first / when Hash / when String` statement? We also need an `else`, which can raise an `ArgumentError`.